From 0922e2a9f4ba38736f65c957a77601510e056e89 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Ol=C5=A1=C3=A1k?= <marek.olsak@amd.com>
Date: Wed, 6 May 2015 18:59:02 +0200
Subject: [PATCH 05/55] amdgpu: fix a valgrind warning

==10307== Conditional jump or move depends on uninitialised value(s)
==10307==    at 0xB9F4CFA: amdgpu_device_initialize (amdgpu_device.c:193)

Reviewed-by: Jammy Zhou <Jammy.Zhou@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Arindam Nath <arindam.nath@amd.com>
---
 amdgpu/amdgpu_device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amdgpu/amdgpu_device.c b/amdgpu/amdgpu_device.c
index b50ce26..048a909 100644
--- a/amdgpu/amdgpu_device.c
+++ b/amdgpu/amdgpu_device.c
@@ -133,7 +133,7 @@ int amdgpu_device_initialize(int fd,
 	int r;
 	int flag_auth = 0;
 	int flag_authexist=0;
-	uint32_t accel_working;
+	uint32_t accel_working = 0;
 
 	*device_handle = NULL;
 
-- 
1.9.1

